notifiers/logNotifier.go

```diff
@@ -0,0 +1,61 @@
+package notifiers
+
+import (
+	"fmt"
+
+	"multiversx/mx-chain-keys-monitor-go/core"
+
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	logger "github.com/multiversx/mx-chain-logger-go"
+)
+
+type logNotifier struct {
+	log logger.Logger
+}
+
+// NewLogNotifier creates a new log notifier instance
+func NewLogNotifier(log logger.Logger) (*logNotifier, error) {
+	if check.IfNil(log) {
+		return nil, errNilLogger
+	}
+
+	return &logNotifier{
+		log: log,
+	}, nil
+}
+
+// OutputMessages will send the output messages to the current logger
+func (notifier *logNotifier) OutputMessages(messages ...core.OutputMessage) {
+	for _, msg := range messages {
+		notifier.output(msg)
+	}
+}
+
+func (notifier *logNotifier) output(message core.OutputMessage) {
+	msg := ""
```

Comment:
Alternatively, `strings.Builder` could have been used. But it's also all right like this (there aren't many concatenations).

Reply:
Message generation could be extracted to a separate function (completely optional).

Reply:
switched to StringBuilder